js/src/dom/class-list.js

```diff
@@ -0,0 +1,27 @@
+function addRemoveClass(add, element, className, classNameList) {
+  if (!classNameList.length) {
+    element.classList[add ? 'add' : 'remove'](className)
+
+    return
+  }
+
+  const classList = [className].concat(classNameList)
+
+  classList.forEach(addClassName => element.classList[add ? 'add' : 'remove'](addClassName))
+}
+
+export function addClass(element, className, ...classNameList) {
```

Comment:
maybe I'm missing something here, but why separate the first className parameter from the spread?
won't `addClass(element, ...classNameList)` work? It will save you having the if statement on line 2

Reply:
nice catch thanks! 👍